R/guide-colorbar.r

```diff
@@ -567,6 +567,57 @@ guide_gengrob.colorbar <- function(guide, theme) {
 #' @rdname guide_colourbar
 guide_colorbar <- guide_colourbar
 
+#' @export
+#' @rdname guide_colourbar
+#'
+#' @param even.steps Should the bin sizes reflect their size or be even across all
+#'   bins? Defaults to `TRUE`
+#' @param show.limits Should labels for the outer limits of the bins be printed?
+#'   Default is `NULL` which makes the guide use the setting from the scale
+guide_coloursteps <- function(even.steps = TRUE, show.limits = NULL, ticks = FALSE, nbin = 100, ...) {
+  guide <- guide_colourbar(raster = FALSE, ticks = ticks, nbin = nbin, ...)
```

Comment:
We really should make this a proper constructor that permits subclassing at some point.

Reply:
agreed. overhaul is for v3.5